config/locales/en.yml

```diff
@@ -203,6 +203,8 @@ en:
     participant_profile:
       blank: "Not registered"
       not_active: "Not active"
+      not_a_participant_profile: "Not a participant profile record"
+      training_complete: "The participant has completed their ECF training"
```

Comment:
✂️

Reply:
Thank you. Remainings from previous versions of the PR.

Reply:
`not_a_participant_profile: "Not a participant profile record"` is still used by the other validator